src/man/firejail.1.in

```diff
@@ -1275,11 +1279,6 @@ Example:
 $ firejail \-\-landlock.fs.read=/ \-\-landlock.fs.write=/home
 \-\-landlock.fs.execute=/usr \-\-landlock.enforce
 #endif
-.TP
-\fB\-\-list
-List all sandboxes, see \fBMONITORING\fR section for more details.
-.br
-
```

Comment:
It looks like this is already sorted.

Also, the move is missing the example below.

Reply:
Nice catch. Fixed!